subsys/bluetooth/services/alexa_gadgets/CMakeLists.txt

```diff
@@ -16,15 +16,15 @@
 # (https://jpa.kapsi.fi/nanopb/download/)
 
 
-# Check system for protoc and nanopb binaries presence.
+# Check system for protoc and nanopb binaries presence, needed to build the gadget profile files.
 # Prefer local variable to make version matching easier
-find_program(PROTOC_BIN protoc HINTS "${PROJECT_SOURCE_DIR}/*/generator-bin/")
-find_program(NANOPB_BIN protoc-gen-nanopb HINTS "${PROJECT_SOURCE_DIR}/*/generator-bin/")
+find_program(PROTOC_BIN protoc HINTS "${CMAKE_SOURCE_DIR}/*/generator-bin/")
+find_program(NANOPB_BIN protoc-gen-nanopb HINTS "${CMAKE_SOURCE_DIR}/*/generator-bin/")
```

Comment:
Now that this is moved out of a sample, then it must be expected the user might use it for more than one sample.
Should we add additional `HINTS` to CMake here ?

What would be good locations: `ZEPHYR_NRF_MODULE_DIR`, `ZEPHYR_NANOPD_MODULE_DIR`, `ZEPHYR_BASE/..`, others ?

Reply:
Good point. Maybe add `ZEPHYR_BASE/../tools` to that list?

Not sure which path `ZEPHYR_NANOPD_MODULE_DIR` refers to though. Do you mean the `ncs/modules/alexa-embedded` directory?

Reply:
`ZEPHYR_NANOPD_MODULE_DIR` would have referred to https://github.com/nrfconnect/sdk-nrf/blob/6c63feff14df3da9d7171f9122d8cc66cd882bc9/west.yml#L121-L124
but looking at it again, that project is not a Zephyr module, so that won't work.

The list was just a list of inspiration :wink: , you shouldn't take all of them.

`ZEPHYR_BASE/../tools` sounds like a good suggestion :+1:

Reply:
Oh, right, I forgot about the nanopb repo. Yeah, neither that or alexa-embedded is a module in the west sense :confused:

I think it makes sense to add `ZEPHYR_BASE/..` and `ZEPHYR_BASE/../tools` to the hints then